### PR TITLE
Add PIWIK_DISABLE_COOKIES.

### PR DESCRIPTION
**Pull request by Anteru**
This option disables cookies -- it's off by default.

See also: https://matomo.org/faq/general/faq_157/

### CODE REVIEW

**Comment by jcassee** [comment]
Thanks, @Anteru!

File: analytical/tests/test_tag_piwik.py [hunk 1]
```diff
@@ -145,3 +145,8 @@ def test_disable_usertrack(self):
         msg = 'Incorrect Piwik user tracking rendering.\nFound:\n%s\nIn:\n%s'
         var_code = '_paq.push(["setUserId", "BDFL"]);'
         self.assertNotIn(var_code, r, msg % (var_code, r))
+
+    @override_settings(PIWIK_DISABLE_COOKIES=True)
+    def test_disable_coolies(self):
```
Comment (Anteru) [inline analytical/tests/test_tag_piwik.py]: and I managed to do a typo here :) Should be `test_disable_cookies`. Sorry about that.

Comment (bittner) [reply analytical/tests/test_tag_piwik.py]: Well, this typo is actually cool! Haha

Functionally, the test is still relevant and correct, I would suppose.

Comment (Anteru) [reply analytical/tests/test_tag_piwik.py]: Of course, it's just the test name.

Comment (jcassee) [reply analytical/tests/test_tag_piwik.py]: Fixed in ba89b80 :-)

**Pull request merged**